accounts-cluster-bench/src/main.rs

```diff
@@ -1423,7 +1423,7 @@ pub mod test {
         solana_native_token::sol_to_lamports,
         solana_poh_config::PohConfig,
         solana_test_validator::TestValidator,
-        spl_token::{
+        spl_token_interface::{
             solana_program::program_pack::Pack,
```

Comment:
We'll probably need to import this ourselves in this file, since the interface doesn't re-export whole crates like the program crate does